nixos/modules/services/hardware/udev.nix

```diff
@@ -160,7 +160,7 @@ let
 
       echo "Generating hwdb database..."
       # hwdb --update doesn't return error code even on errors!
-      res="$(${pkgs.buildPackages.udev}/bin/udevadm hwdb --update --root=$(pwd) 2>&1)"
+      res="$(${config.systemd.package}/bin/systemd-hwdb --root=$(pwd) update 2>&1)"
```

Comment:
will break cross because systemd.package will be for host

Reply:
Thanks for the review. I'll update to use pkgs.buildPackages.systemd, rebase to master branch, and re-test.

Reply:
@Artturin I made the requested change and moved to master. Thanks for your help!